sql/core/src/test/scala/org/apache/spark/sql/DataFrameJoinSuite.scala

```diff
@@ -115,7 +115,7 @@ class DataFrameJoinSuite extends QueryTest with SharedSQLContext {
       Row("1", 1) :: Row("2", 1) :: Row("3", 1) :: Nil)
   }
 
-  test("[SPARK-6231] join - self join auto resolve ambiguity") {
+  ignore("[SPARK-6231] join - self join auto resolve ambiguity") {
```

Comment:
With this change, I'm not sure this "auto resolve ambiguity" feature is useful anymore.